tests/check_framework/test_urls.py

```diff
@@ -35,3 +35,9 @@ def test_name_with_colon(self):
         self.assertEqual(warning.id, 'urls.W003')
         expected_msg = "Your URL pattern '^$' [name='name_with:colon'] has a name including a ':'."
         self.assertIn(expected_msg, warning.msg)
+
```

Comment:
I think you need to decorate the test method with `@override_settings` so that the original value value of `ROOT_URLCONF` is restored at the end of the test.